### ISSUE TITLE
Use correct field ordering

### ISSUE THREAD

**Issue by turicas** [turicas/rows#239]
Code to reproduce:

```python
from collections import OrderedDict

import rows


data = [{'a': 1, 'b': 2},
        {'a': 3, 'b': 4}, ]
fields = OrderedDict([('b', rows.fields.IntegerField),
                      ('a', rows.fields.IntegerField)])
table = rows.import_from_dicts(data, fields=fields)
final_data = list(table)
assert final_data[0].a == 1
assert final_data[0].b == 2
assert final_data[1].a == 3
assert final_data[1].b == 4

# problem: inside create_table when use `fields` with `dicts`
```

**Comment by turicas** [comment]
I've started working on this branch: https://github.com/turicas/rows/tree/fix/239-fields-ordering.

**Comment by turicas** [comment]
Related to #238

**Comment by marcosvbras** [comment]
I will help with this issue

**Comment by turicas** [comment]
Was not completely fixed on  8a992ab. Working on https://github.com/turicas/rows/tree/fix/239-fields-ordering
